JobConfig/ensemble/python/normalizations.py

```diff
@@ -174,13 +174,14 @@ def corsika_offspill_normalization(livetime, run_mode = '1BB'):
 def rpc_normalization(onspilltime, tmin, internal, emin, run_mode = '1BB'):
   POT = getPOT(onspilltime, run_mode)
   # hack: --> will come from new table eventually
-  npistops = 1287106 # from above
-  target_stopped_pi_per_POT =  0.01337 * 0.1670 # from above
-  time_eff = 83146/npistops # from above
-  
-  total_sum_of_weights = 1148 # from filter - TODO
-  selected_sum_of_weights = 0.178864 # from filter - TODO
-
+  npistops = 41798702 ## from above (MDC2020aw)
+  target_stopped_pi_per_POT =  0.003924801 * 0.522483775 # from above (MDC2020ak)
+  npifilter = 4378518 # from above (MDC2020aw)
+  time_eff = npifilter/npistops  
+  total_sum_of_weights =  37468# from filter
+  selected_sum_of_weights =  5.7055#from filter
+
+  gen_factor = 50 # from Michael (for the new sampling factors)
```

Comment:
Can you add to the comment how these numbers were extracted? Ie > samweb dh.dataset="....", etc
We need a long-term plan how these numbers can be determined from production output directly.